python/samples/concepts/agents/bedrock_agent/README.md

```diff
@@ -0,0 +1,45 @@
+# Concept samples on how to use AWS Bedrock agents
+
+## Pre-requisites
+
+1. You need to have an AWS account and [access to the foundation models](https://docs.aws.amazon.com/bedrock/latest/userguide/model-access-permissions.html)
+2. [AWS CLI installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [configured](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration)
+
+## Samples
+
+| Sample | Description |
```

Comment:
We have a main `samples/concepts`[README.md](https://github.com/microsoft/semantic-kernel/blob/main/python/samples/concepts/README.md) that contains details about all of our concept samples. Would be good to add something about the `bedrock` directory.